core/integration/storagetest/mutation_logs.go

```diff
@@ -64,11 +64,17 @@ func (mutationLogsTests) TestReadLog(ctx context.Context, t *testing.T, newForTe
 	m, done := newForTest(ctx, t, directoryID, logID)
 	defer done(ctx)
 	// Write ten batches.
+	var lastTS time.Time
 	for i := byte(0); i < 10; i++ {
 		entry := &pb.EntryUpdate{Mutation: &pb.SignedEntry{Entry: mustMarshal(t, &pb.Entry{Index: []byte{i}})}}
-		if _, err := m.Send(ctx, directoryID, logID, entry, entry, entry); err != nil {
+		ts, err := m.Send(ctx, directoryID, logID, entry, entry, entry)
```

Comment:
Optional side-quest: You could use `lastTS` here instead of introducing `ts` and I think it all checks out.

Reply:
True, but then I'd have to side declare err. Not sure which one is cleaner.